nimbus/src/client/http_client.rs

```diff
@@ -227,6 +236,7 @@ mod tests {
                 "proposedEnrollment": 7,
                 "referenceBranch": "control",
                 "probeSets": [],
+                "featureIds": ["some_switch"],
```

Comment:
My inference from brief discussions with Kate is that this is not currently supported, the reason the schema has already been updated to use an array is more along the lines of future-proofing.

I don't have a clear line-of-sight on how multiple features would work either.  A big part of that is due to me being new to the SDK codebase, but I am also a bit concerned that the details of making multiple features work may have hard-to-resolve complexity.

All that said, I don't have any plans of attempting to address multiple-feature support/enrollment in this PR at all (or even anytime soon).  I've added the array to match the existing schema being output by experimenter, but if we wanted to change that, I suspect it's not too late.

@k88hudson, @travis79 any thoughts here?

Reply:
While it does complicate things to have to check that only one experiment is active per feature, one of the original goals of Nimbus was to enable multiple experiments to be running at the same time.  Rather than limiting this to one experiment per client (which I agree, would make things a lot easier), we really do need to ensure that there is only one experiment per feature, but whether or not this happens in the SDK or somewhere before the experiment is launched, I am not for certain.

Reply:
> It is significantly easier if only one experiment is active per user.

Yeah, definitely need to allow more than 1 experiment per client.

Re: whether we enforce at the server v.s. the client, ideally the server – the only exception is that sometimes you want to roll-over and re-target the unenrolled users same bucket range if the experiments have different targeting (e.g. one is only first run users, one is any fxa user or something like that)

Reply:
Thanks for the clarification.  I think this text doesn't directly effect this PR, so marking it as resolved.